lib/parsers/txt.mjs

```diff
@@ -263,16 +263,19 @@ export async function parse (rawText, filename) {
       }
 
       // Searching for copyright line
-      const match = [...line.matchAll(licenseDeclarations.copyright_section6)]
+      const copyrightLineMatch = [...line.matchAll(licenseDeclarations.copyright_section6)]
```

Comment:
Correct me if I am wrong, but I think that you should search for this copyright not in lines but in normalized text before loop, because text, that you are looking for can be separated on different lines. And now I can`t make validator don`t complain about this issue